xtask/src/main.rs

```diff
@@ -644,9 +644,7 @@ fn lint_packages(workspace: &Path, args: LintPackagesArgs) -> Result<()> {
 fn lint_package(path: &Path, args: &[&str]) -> Result<()> {
     log::info!("Linting package: {}", path.display());
 
-    let mut builder = CargoArgsBuilder::default()
-        .toolchain("esp")
-        .subcommand("clippy"); // TODO: Is this still actually required?
+    let mut builder = CargoArgsBuilder::default().subcommand("clippy"); // TODO: Is this still actually required?
```

Comment:
I think that comment was regarding the "esp" toolchain?

Reply:
Removed the comment, we are already using the [`esp` channel in the CI](https://github.com/esp-rs/esp-hal/pull/1881/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR116) and I though it would be helpful to be able to use other channels, [like `nightly`](https://github.com/esp-rs/esp-hal/pull/1881/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR183)